lib/pact/helpers.rb

```diff
@@ -29,5 +29,17 @@ def each_like content, options = {}
       Pact::ArrayLike.new(content, options)
     end
 
+    def like_uuid uuid
+      Pact::Term.new(generate: uuid, matcher: /[0-9a-f]{8}(-[0-9a-f]{4}){3}-[0-9a-f]{12}/)
+    end
+
+    def like_datetime datetime
+      Pact::Term.new(generate: datetime, matcher: /\d{4}\-\d{2}\-\d{2}T\d{2}:\d{2}:\d{2}([+-]\d{2}:\d{2})?/)
```

Comment:
Agreed, updated Regex for date and datetime matchers, not sure that we need to be more strict hire. New Regex not ideal, but much better :)